MobileFoodSchedules/MobileFoodSchedules/MobileFoodSchedule.json

```diff
@@ -0,0 +1,219 @@
+﻿{
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "type": "array",
+  "items": {
+    "$ref": "#/definitions/WelcomeElement"
+  },
+  "definitions": {
+    "WelcomeElement": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "dayorder": {
+          "type": "string",
+          "format": "integer"
+        },
+        "dayofweekstr": {
+          "$ref": "#/definitions/Dayofweekstr"
+        },
+        "starttime": {
+          "$ref": "#/definitions/Time"
+        },
+        "endtime": {
+          "$ref": "#/definitions/Time"
+        },
+        "permit": {
+          "type": "string"
```

Comment:
As long as you're adding the schema, you might as well add some validation rules to it... like the maximum value of a number, the max length of a String, etc.